omod/src/main/resources/config.xml

```diff
@@ -96,6 +96,13 @@
 		</description>
     </globalProperty>
     <globalProperty>
+        <property>aijar.nhpi</property>
+        <defaultValue>HFT2ZBPE8</defaultValue>
```

Comment:
@slubwama The default has to be a blank value otherwise all data will be submitted as this NPI